lib/ZwaveClient.ts

```diff
@@ -207,7 +207,9 @@ export type SensorTypeScale = {
 
 export type AllowedApis = typeof allowedApis[number]
 
-const ZWAVEJS_LOG_FILE = utils.joinPath(storeDir, 'zwavejs_%DATE%.log')
+const logDir = process.env.ZWAVEJS_LOG_DIR || storeDir
+
+const ZWAVEJS_LOG_FILE = utils.joinPath(logDir, 'zwavejs_%DATE%.log')
```

Comment:
I would use LOGS instead of LOG, also no need to use a const for it it's quite easy to understand even with a single code line : 

```suggestion
const ZWAVEJS_LOG_FILE = utils.joinPath(process.env.ZWAVEJS_LOGS_DIR || storeDir, 'zwavejs_%DATE%.log')
```